minio/compat.py

```diff
@@ -93,21 +93,23 @@
 
 numeric_types = (int, long, float)
 
-def urlencode(resource):
+
```

Comment:
Can you leave a note for the `deprecation` as a comment.. Just to record the change?

Reply:
Sure - I'll add as a comment rather than in the docstring so it isn't visible to end users.